### PR TITLE
Add Fulcro config

### PR DESCRIPTION
**Pull request by holyjak**
This includes two separate commits:

1. b59650e `Dev support` - shadow-powered dev repl  - see the commit for details. This is a proposal and may be dropped
2. `Initial support for Fulcro` - see the commit for details

### CODE REVIEW

**Comment by holyjak** [comment]
Made into draft - need to port fully two hooks macros.

File: src/sci/configs/fulcro/routing/dynamic_routing.cljs [hunk 1]
```diff
@@ -0,0 +1,89 @@
+(ns sci.configs.fulcro.routing.dynamic-routing
+  (:require [cljs.analyzer :as ana]
```
Comment (borkdude) [inline src/sci/configs/fulcro/routing/dynamic_routing.cljs:2]: Are you sure you want to pull in the CLJS analyzer for just throwing an error? It seems a waste of bundle size

Comment (holyjak) [reply src/sci/configs/fulcro/routing/dynamic_routing.cljs]: No, I do not. Good point, thx! Fixed now.

**Comment by holyjak** [comment]
Made to draft, while troubleshooting state management in Fulcro.

**Pull request merged**